direct/src/directnotify/Notifier.py

```diff
@@ -42,12 +47,12 @@ def __init__(self, name, logger=None):
             self.__logger = logger
 
         # Global default levels are initialized here
-        self.__info = 1
-        self.__warning = 1
-        self.__debug = 0
-        self.__logging = 0
+        self.__info = True
+        self.__warning = True
+        self.__debug = False
+        self.__logging = False
 
-    def setServerDelta(self, delta, timezone):
+    def setServerDelta(self, delta: SupportsRound[SupportsInt], timezone: int) -> None:
```

Comment:
Can we just make this a float? That's probably what's intended to be passed here. Would that prevent passing an int?

Reply:
Annotating a parameter as a float is special-cased by PEP 484 to also accept ints, so that would work. I doubt anyone will be passing custom numeric types to this function anyway.

Reply:
Yes, let's just do float.